schemas/maas-backend/customers/verification/initiate/response.json

```diff
@@ -0,0 +1,81 @@
+{
+  "$id": "http://maasglobal.com/maas-backend/customers/verification/initiate/response.json",
+  "description": "MaaS customer verification initiate",
+  "type": "object",
+  "properties": {
+    "customer": {
+      "type": "object",
+      "properties": {
+        "verification": {
+          "type": "array",
+          "minItems": 1,
+          "items": {
+            "type": "object",
+            "properties": {
+              "id": {
+                "type": "string",
+                "minLength": 1,
+                "maxLength": 255,
+                "description": "Unique ID"
+              },
+              "identityId": {
+                "$ref": "http://maasglobal.com/core/components/units.json#/definitions/identityId"
+              },
+              "details": {
+                "type": "object",
+                "properties": {
+                  "url": {
+                    "type": "string"
+                  },
+                  "sessionToken": {
+                    "type": "string"
+                  }
+                },
+                "required": ["url", "sessionToken"],
+                "additionalProperties": true
+              },
+              "validFrom": {
+                "type": "string"
+              },
+              "validTo": {
+                "type": "string"
+              },
+              "type": {
+                "type": "string"
+              },
+              "stateLog": {
+                "type": "array",
+                "minItems": 1,
+                "items": {
+                  "type": "object",
+                  "properties": {
+                    "status": {
+                      "type": "string"
+                    },
+                    "timestamp": {
+                      "$ref": "http://maasglobal.com/core/components/units.json#/definitions/time"
+                    }
+                  },
+                  "required": ["status", "timestamp"]
+                }
+              },
+              "created": {
+                "$ref": "http://maasglobal.com/core/components/units.json#/definitions/time"
+              },
+              "modified": {
+                "$ref": "http://maasglobal.com/core/components/units.json#/definitions/time"
+              },
+              "agencyId": {
+                "type": "string"
```

Comment:
There should be agencyId definition in core/components

Reply:
Well spotted! Fixed!